app/models/potential_match.rb

```diff
@@ -235,12 +235,14 @@ def self.progress(collection)
     collection_byline_key = byline_key(collection)
     unless REDIS_GENERAL.exists(collection_byline_key)
       score = 0
-      collection.signups.pseud_only.find_each do |pseud|
+      collection.signups.pseud_only.each do |pseud|
         REDIS_GENERAL.zadd collection_byline_key, score, pseud.byline
         score += 1
       end
     end
-    progress = (REDIS_GENERAL.zrank(collection_byline_key, current_byline)  * 100)/REDIS_GENERAL.zcount(collection_byline_key, 0, "+inf")
+    rank = REDIS_GENERAL.zrank(collection_byline_key, current_byline)
+    return -1 if rank = nil # something's wrong
```

Comment:
Shouldn't this be `if rank == nil`, or better yet `if rank.blank?`
On this occasion the Hound was useful - the assignment is useless because the variable gets reassigned. >.>